### DISCUSSION TITLE
Mia Native Config Refresh for Mia Plugins

### DISCUSSION THREAD

**Discussion by zachhutchins** [mia-platform/community#487] [Q&A]
### What product are you having troubles with?

Console

### What Console version are you using?

1.98.6

### Description

We have a need to refresh applications configurations when changes are made to config maps. Most applications typically support some kind of hot reload to configurations. This happens internally through file watches or exposing an endpoint .e.g `/-/refresh`.

I do not see anything documented that suggests this kind of feature exists for apps Authorization Service, or JWT Validator as an example of plugins that we need it for.

**Comment by fredmaggiowski** [comment]
HI @zachhutchins 

this seems like a nice feature to have implemented in our services; do you need it for something else other than the two you specified?

I'd ask you to contact your Mia-Platform referent with this feature request so that we can consider and plan it :)
